scripts/build-ssr.js

```diff
@@ -24,10 +27,20 @@ const { port } = require('../context');
     await run(webpack(serverConfig));
     await copyPublicFiles();
 
-    console.log(green('Sku build complete!'));
-  } catch (e) {
-    console.error(red(e));
+    const timeTaken = performance.now();
+    track.timing('build_ssr', timeTaken, { status: 'success' });
```

Comment:
Using a seperate key for build_ssr will make it hard to report on 'how many builds of any kind have happened'.
Would it make sense to call the key 'build' and move ssr into a tag?
```suggestion
    track.timing('build', timeTaken, { status: 'success', type: 'ssr' });
```

Reply:
Addressed this for both build and start